.travis.yml

```diff
@@ -102,7 +102,7 @@ matrix:
         # time. We don't expect any of these to fail in master for cron jobs.
         - os: linux
           stage: Tests with other Python/Numpy versions
-          env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.9  KEYRING_VERSION='<12.0'
+          env: PYTHON_VERSION=2.7 NUMPY_VERSION=1.10  KEYRING_VERSION='<12.0'
```

Comment:
If it can help the travis error was something like :
``ValueError: numpy.dtype has the wrong size, try recompiling.``
and occured when importing CircleSkyRegion, PolygonSkyRegion from regions in the core.py.